5_recognition.ipynb

```diff
@@ -205,11 +368,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "画像データと同様に、以下の命令で左上のピクセル値にアクセスできます。単位はメートルです。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Similer to the image data, we can access the depth value in the upper left with the following command. The unit is meters:"
```

Comment:
"Similar to the image data, we can access the depth value in the upper left with the following command. The unit is meters:"